sympy/core/power.py

```diff
@@ -1628,7 +1629,7 @@ def e2int(e):
         # either b0 is bounded but neither 1 nor 0 or e is infinite
         # b -> b0 + (b - b0) -> b0 * (1 + (b/b0 - 1))
         o2 = order*(b0**-e)
-        z = (b/b0 - 1)
+        z = (b/b0 - 1).simplify()
```

Comment:
Is it necessary to apply the full `simplify`?

Reply:
Though for some expressions it might not be necessary to apply full simplify, but for all in general a full simplify will ensure that z is as much simplified as possible, thus minimum possible terms will be printed

Reply:
`g = self._taylor_term(i, z, g)`, so z must be as much simplified as possible

Reply:
Moreover the fact that no testcase is timing out due to the usage of simplify(), I think it is just fine to use it for now.

Reply:
Operations on AccumBounds seem to have some issues that should be fixed. I think that the following could be used to circumvent those, at least provisionally:

    if isinstance(b0, AccumBounds):
        z = b/b0 - 1
    else:
        z = (b - b0)/b0

I would want to avoid `simplify()` in this context though some additional simplification should be added to `Order`.